charts/veritable-ui/templates/_helpers.tpl

```diff
@@ -0,0 +1,188 @@
+{{/*
+Return the proper veritable-ui image name
+*/}}
+{{- define "veritable-ui.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper init container image name
+*/}}
+{{- define "veritable-ui.initDbCreate.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.initDbCreate.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "veritable-ui.imagePullSecrets" -}}
+{{- include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.initDbCreate.image ) "global" .Values.global) -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "veritable-ui.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (include "common.names.fullname" .) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the Postgresql hostname
+*/}}
+{{- define "veritable-ui.databaseHost" -}}
+{{- ternary (include "veritable-ui.postgresql.fullname" .) .Values.externalDatabase.host .Values.postgresql.enabled | quote -}}
+{{- end -}}
+
+{{/*
+Return the company profile API key secret name
+*/}}
+{{- define "veritable-ui.companyHouseSecret" -}}
+{{- ternary (include "veritable-ui.companyHouse.fullname" .) .Values.companyHouse.secret .Values.postgresql.enabled | quote -}}
+{{- end -}}
```

Comment:
Don't understand what this is going to achieve.

Reply:
put in `companyHouseSecret` variable for later usage, did follow postgres "existing secret config"

Reply:
this is also should have been updated